allure-ruby-commons/lib/allure_ruby_commons/model/parameter.rb

```diff
@@ -3,13 +3,15 @@
 module Allure
   # Allure model parameter object
   class Parameter < JSONable
-    def initialize(name, value)
+    def initialize(name, value, excluded: false, mode: 'default')
```

Comment:
Is it documented somewhere what are the actual acceptable values for `mode`? It would be good to create constants with acceptable modes similar to how `ContentType` is defined for attachments.